### PR TITLE
Topic 01 excercise 04

### PR DESCRIPTION
**Pull request by vvale93**

### CODE REVIEW

File: Topic-00-Excercise-03/First-CSS-Excercise/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    <li class="navBarItem"><a href="">Awesome link 5</a></li>
+                    <li class="navBarItem"><a href="">Awesome link 6</a></li>
+                    <li class="navBarItem"><a href="">About us</a></li>
+                    <li class="navBarItem"><a href="">Contact</a></li>
+            </nav>
+            <section class="content">
+                <header class="header-content">Awesome header</header>
+                <article class="article"><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam varius magna eu hendrerit sagittis. Quisque ex orci, suscipit ut tortor quis, pharetra facilisis ipsum. Nulla ex arcu, tincidunt eu imperdiet at, condimentum ut metus. Ut quis tristique odio. Etiam in pretium ligula. Integer vehicula sollicitudin eros porta mollis. Donec quis neque eget ex condimentum congue sit amet a metus. Quisque tempor dignissim elit, in semper erat blandit quis. Pellentesque ac hendrerit erat. Donec non finibus libero. Maecenas id libero vel nisl molestie accumsan. Phasellus a iaculis quam. In maximus diam orci, eget lobortis lorem auctor vitae.</p></article>
```
Comment (DemianMoschin-GLB) [inline Topic-00-Excercise-03/First-CSS-Excercise/index.html]: Please add tags in separate lines

File: Topic-00-Excercise-03/First-CSS-Excercise/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+.navBarItemContainer
+{
+    
+    list-style-type: none;
+}
+.body
+{
+    font-weight: bold;
+    font-family: 'Patua One', cursive;
+}
```
Comment (DemianMoschin-GLB) [inline Topic-00-Excercise-03/First-CSS-Excercise/style.css]: New line here required

File: Topic-00-Excercise-03/Second-CSS-Excercise/index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <div class="column">
+                <div class="screen">
+                    <p class="title">
+                        Beautiful sunset
+                    </p>
+                </div>
+                <img src="photo6.jpg" alt="beautiful sunset">
+            </div>
+        </div>
+</body>
```
Comment (DemianMoschin-GLB) [inline Topic-00-Excercise-03/Second-CSS-Excercise/index.html]: Minor: Indent

File: Topic-01-Excercise-04/script.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                xhr.send();
+            });
+        };
+        request(config)
+            .then(data => {
+                let response = JSON.parse(data); 
+                let items = response.items;
+                for(let i = 0; i<items.length; i++)
+                {
+                document.getElementById("results").innerHTML += "<li><a href=\"https://github.com/"+items[i].full_name+"\">"+items[i].name+"</a></li>";
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-04/script.js]: Indent

Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-04/script.js]: Please avoid doble-quotes. Use single-quotes instead. 
Also: use Template literals if you want to concat strings

File: Topic-01-Excercise-04/style.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    width: 30%;
+    box-sizing: border-box;
+}
+
+.section
+{
+    float: left;
+    width: 70%;
+    box-sizing: border-box;
+}
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-04/style.css]: add new line after

**Comment by vvale93** [comment]
resolved minors and implemented template literals

File: Topic-00-Excercise-03/First-CSS-Excercise/style.css (hunk 2 above)
Comment (vvale93) [reply Topic-00-Excercise-03/First-CSS-Excercise/style.css]: done

File: Topic-01-Excercise-04/style.css (hunk 5 above)
Comment (vvale93) [reply Topic-01-Excercise-04/style.css]: done

**Pull request merged**